src/main/webapp/creatives/maps/interaction.js

```diff
@@ -0,0 +1,45 @@
+
+import {generateSessionId} from '../maps/utils.js';
+
+const URL = {
+  INITIALIZE: "initialize",
+  FIND_NEAREST_LOCATION: "find-location",
+  GRANT_LOCATION_DATA: "grant-location",
+  MAP_INTERACT: "map-interact",
+  SKIP_TO_CONTENT: "skip-to-content",
+  RETURN_TO_AD: "return-to-ad"
+}
+
+class User {
```

Comment:
I like how you structured this class I just think a better name could be used for the class, such as userSession

Reply:
Renamed it!